index.html

```diff
@@ -2,13 +2,539 @@
+        fill="#fff"
+      />
+    </symbol>
+  </svg>
+    <!--Block 1-->
+    <div class="block-one__background">
+      <div class="container">
+        <div class="block-one__header">
+          <div class="block-one__header--left">
+            <a href="#">
+              <svg class="block-one__header--left--svg">
+                <use href="#trip-header-svg"/>
+              </svg>
+            </a>
+          </div>
+          <div class="block-one__header--right">
+            <div>
+              <a
+                class="block-one__header--right--href block-one__header--right--href--stays"
+                href="#"
+                >Stays</a
+              >
+              <a
+                class="block-one__header--right--href block-one__header--right--href--attractions"
+                href="#"
+                >Attractions</a
+              >
+            </div>
+            <div class="block-one__header--right--logotip">
+              <a href="#">
+                <svg class="block-one__header--right--logotip--night">
+                  <use href="#logotip__night"/>
+                </svg>
+              </a>
+              <a href="#">
+                <svg class="block-one__header--right--logotip--account">
+                  <use href="#logotip__account"/>
+                </svg>
+              </a>
+              <a href="#">
+                <svg class="block-one__header--right--logotip--menu phone">
+                  <use href="#logotip__menu"/>
+                </svg>
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="block-one__page">
+          <div class="block-one__page--header">
+            <p>Discover stays</p>
+            <p>to live, work or just relax</p>
+          </div>
+          <div class="block-one__page--underform">
+            <div>Your destination or hotel name</div>
+            <div>Check-in — Check-out</div>
+          </div>
+          <form class="block-one__page--form">
+            <div class="block-one__page--form--filters">
+              <input
+                class="block-one__page--input--city-desktop desktop"
+                type="text"
+                placeholder="New York"
+              />
+              <!--For mobile phone-->
+              <div class="block-one__page--form--city-phone phone">
+                <img
+                  class="search-img phone"
+                  src="./images/seach.png"
+                  alt="seach"
+                />
```

Comment:
You added the image file filled with white for padding 16px in the input. It doesn't make sense.

Reply:
I understund. I corrected it.

Reply:
and I corrected position circle on block-3.